backend/main.py

```diff
@@ -1,15 +1,98 @@
-from typing import Union
+import os
+
+from fastapi import FastAPI, Body
+from fastapi.responses import RedirectResponse
+
+from pytrello2 import TrelloClient
+
+# Constants for todo-trello
+BOARD = "personal"
+LISTS = ["urgent", "important", "sometime"]
+LIST = "doing"
+
+# Environment variables
+API_KEY = os.environ.get("TRELLO_API_KEY")
+TOKEN = os.environ.get("TRELLO_TOKEN")
 
-from fastapi import FastAPI
 
 app = FastAPI()
 
 
+def get_trello_client(api_key: str, token: str) -> TrelloClient:
+    return TrelloClient(api_key, token)
+
+client = get_trello_client(API_KEY, TOKEN)
+
+
 @app.get("/")
-def read_root():
-    return {"Hello": "World"}
+async def root():
+    # Retrieve all boards
```

Comment:
Is this board the same concept as the list of todos? so the tasks below on line 52 will act as our todo item?

Reply:
There is one board called `personal`, one list on the board called `doing` and there will be multiple tasks on that list.